yt/geometry/tests/test_particle_octree.py

```diff
@@ -407,7 +409,8 @@ def test_bitmap_select():
 
 
 @requires_file("TNGHalo/halo_59.hdf5")
-def test_ewah_write_load(tmp_path):
+def test_ewah_write_load():
+    tmp_path = Path(tempfile.mkdtemp())
```

Comment:
from `tempfile`'s doc
> The user of [mkdtemp()](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp) is responsible for deleting the temporary directory and its contents when done with it.

I'd rather we keep the `pytest` form, but if you don't want to, I'd advise using a context manager (`tempfile.TemporaryDirectory`) to avoid side effects